contract/audit_log.sol

```diff
@@ -0,0 +1,31 @@
+pragma solidity >=0.6.0 <0.7.0;
+
+/**
+ * @title Storage
+ * @dev Store & retrieve value in a variable
+ */
+contract audit_log{
+    // The table for mapping DB table name and address which save the the audit log of DB
+    // operations.
+    mapping(string => address) auditLogMap;
+
+    /**
+     * @dev Add a new DB table name and corresponding ethereum public key
+     */
+    // FIXME We need to implement a mechanism to avoid tableName conflicting among different organizations
```

Comment:
How about introducing a hashtable and/or dictionary to abstract the naming?

Reply:
`mapping` is a hashtable in solidity.  Do you mean that we need another mapping to construct a  layer hashtable structure?